20_beta_diversity.Rmd

```diff
@@ -16,18 +16,27 @@ knitr::opts_chunk$set(
 
 # Community Similarity {#community-similarity}
 
-Whereas alpha diversity focuses on community variation within a community
-(one sample), beta diversity quantifies the dissimilarity between communities
-(multiple samples). In microbiome research, the most popular metrics of beta
+Beta diversity quantifies the dissimilarity between communities (multiple
+samples), as opposed to alpha diversity which focuses on variation within a
+community (one sample). In microbiome research, commonly used metrics of beta
 diversity include the Bray-Curtis index (for compositional data), Jaccard index
-(for presence / absence data, ignoring abundance information), Aitchison distance
+(for presence/absence data, ignoring abundance information), Aitchison distance
 (Euclidean distance for clr transformed abundances, aiming to avoid the
 compositionality bias), and the Unifrac distance (that takes into account the
 phylogenetic tree information). Notably, only some of these measures are actual
 _distances_, as this is a mathematical concept whose definition is not satisfied
 by certain ecological measure, such as the Bray-Curtis index. Therefore, the terms
 dissimilarity and beta diversity are preferred.
 
+| Method description         | Assay type          | Beta diversity metric |
+|:--------------------------:|:-------------------:|:---------------------:|
+| Quantitative profiling     | Absolute counts     | Bray-Curtis           | 
+| Relative profiling         | Relative abundances | Bray-Curtis           |
+| Aitchison distance         | clr                 | Euclidean             |
+| Robust Aitchison distance  | rclr                | Euclidean             |
+| Present/Absence similarity | Absolute counts     | Jaccard               |
+| Phylogenetic distance      | Absolute counts     | Unifrac               |
+
```

Comment:
@antagomir Can you quickly check if these are correct

Reply:
Present/Absence -> Presence/Absence
-> and here both absolute or relative counts are ok, they will yield the same result

(r)Aitchison can also be done with "aitchison" and "no transformation" (i.e. the clr is already part of the distance itself). Both are available in mia, and should give identical result. It might be worth mentioning (comment field?) that both options are available.

Phylogenetic distance: hmm at least I would rarify the data; although that might be useful also for Bray-Curtis with relative abundances (albeit controversial..)